src/imageio/format/avif.c

```diff
@@ -52,16 +52,23 @@ enum avif_tiling_e {
   AVIF_TILING_OFF
 };
 
+enum avif_color_mode_e {
```

Comment:
Just see that now, dt style is to have the { under enum, typedef, switch, if...

Can you please fix this as part of this PR ? TIA.

Reply:
Done.